apps/src/code-studio/dancePartySongArtistTags.js

```diff
@@ -35,13 +35,23 @@ export const SongTitlesToArtistTwitterHandle = {
   wenospeakamericano_yolandabecool: 'YolandaBeCool',
   ymca_villagepeople: 'VillagePeople',
   firework_katyperry: 'KatyPerry',
+  //2019 Songs
+  calma_pedrocapo: 'Pedrocapo',
+  highhopes_panicatthedisco: 'PanicAtTheDisco',
+  ificanthaveyou_shawnmendes: 'ShawnMendes',
+  introtoshamstep_47SOUL: '47soulofficial',
+  kidzbop_ificanthaveyou_shawnmendes: 'KidzBop',
+  neverreallyover_katyperry: 'KatyPerry',
+  occidentalview_francescogabbani: 'FrankGabbani',
+  oldtownroadremix_lilnasx: 'LilNasX',
+  starships_nickiminaj: 'NickiMinaj',
+  sucker_jonasbrothers: 'JonasBrothers',
```

Comment:
Are any of the potentials listed below still options? Should we delete them?

Reply:
We actually ended up getting one of last year's potentials this year, FWIW. Not sure if any of these are still in consideration. I'll check with Min

Reply:
ok cool.  there's no real harm in keeping them, mostly just curious.